### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Queues_ViewMessage_Content on Hover or focus : The tool-tip content for several controls such as "View Message" is not hoverable using mouse pointer.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3506]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.13;#MAC;#A11ySev2;#Benchmark;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Queues" of the selected subscription and expand it.
5. Navigate to any queue and click on it.
6. Hover over "View message" button using mouse.
7. Check whether appeared tooltip content is hoverable using mouse or not.

### Actual:
When users navigate to the several controls such as "View Message" the tool-tip content present for each control is not hoverable using mouse pointer.

### Expected:
The tool-tip content for each of the controls such as "View Message" should be hoverable using mouse pointer.

### User Impact:
If the tooltip content is not hoverable using mouse then low vision users will have difficulty in getting the information that is available on the tooltip content.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/Eejp6jBMu4lMnELtoTmwUuwB3uK_wOhAVBXzWfI1bKApPg?e=8FToND

**Comment by JasonYeMSFT** [comment]
Our tooltips are provided by the browser through the "title" attribute. This is an exception of this accessibility rule and there is no way we can fix it.

**Comment by JasonYeMSFT** [comment]
@MayuriNaik Could you please verify and close this issue if it is fixed?

**Comment by MayuriNaik** [comment]
@JasonYeMSFT Closing this bug per the discussion with the SMEs. This is an exception since the tooltips are provided by the browser’s title attribute.

**Comment by Ashutoshsahoo164** [comment]
GitHubTags : #Closed
